images/Dockerfile

```
@@ -472,6 +472,9 @@ FROM pelican-software-base AS director
 ARG TARGETOS TARGETARCH
 COPY --from=pelican-build /pelican-build/dist/${TARGETOS}_${TARGETARCH}/pelican_${TARGETOS}_${TARGETARCH}/pelican /usr/local/bin/pelican
 COPY images/entrypoint.sh /entrypoint.sh
+COPY scripts/geoquery.py /usr/local/sbin/geoquery
+RUN dnf install -y python3-pip
+RUN python3 -m pip install geoip2
```

```suggestion
RUN dnf install -y python3-pip \
    && python3 -m pip install geoip2
```

To reduce the number of layers / container size